.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseMergedExtractor.cs

```diff
@@ -110,8 +110,17 @@ private void AddTo(List<ExtractResult> dst, List<ExtractResult> src, string text
                     //}
                     if (dst[i].IsOverlap(result))
                     {
+                        int nextCursor = i + 1;
+                        var isCrossOverlap = false;
+                        if (nextCursor < dst.Count)
+                        {
+                            if (dst[nextCursor].IsCrossOverlap(result))
+                            {
+                                isCrossOverlap = true;
+                            }
+                        }
                         isFound = true;
-                        if (dst[i].IsCover(result))
+                        if (dst[i].IsCover(result) && !isCrossOverlap)
```

Comment:
It looks like this change is also breaking other test cases.
If you look at the build fail log, this now fails:
Failed   MergedExtractor (Data Row 3)
Error Message:
 Assert.AreEqual failed. Expected:<1>. Actual:<2>. Input: "this is before tomorrow 4pm "

Please always run all test cases for all platforms (dotnet, javascript, python) before creating the PR.